modules/pxp-module-puppet

```diff
@@ -102,6 +102,18 @@ def make_error_result(exitcode, error_type, error_message)
   return result
 end
 
+def parse_report(filename)
+  # Read the report and drop Ruby objects first.
+  # We can't parse the Ruby objects (because we don't have Puppet loaded)
+  # and don't need that data.
+  data = YAML.parse_file(filename)
+  data.root.each do |o|
```

Comment:
Updated the comment; Ruby's YAML defers to a Psych implementation and took a little digging to find the right classes/methods.